metadata.rb

```diff
@@ -45,7 +45,7 @@
 supports 'suse'
 supports 'ubuntu'
 
-depends 'swap', '~> 0.3'
+depends 'swap'
```

Comment:
Why not update the condition instead of removing it? Any reason? :confused:

Reply:
sure I can do that.